### DISCUSSION TITLE
make smime.p7n readable

### DISCUSSION THREAD

**Discussion by Hubhubhurra** [the-djmaze/snappymail#1181] [Ideas]
Hallo, 
some email-clients merge a digital sign public certificate together with the message-body  to a single zip which snappy only show as attachment smime.p7n. No text is readable. To read it is necessary download the attachment and unzip it.
Thunderbird and other client-programms can show the message directly and it was a great think if snappy can this too.

with regards

**Comment by the-djmaze** [comment]
Now you can https://github.com/the-djmaze/snappymail/releases/tag/v2.35.0
